### ISSUE TITLE
508-defect-3 [COGNITION]: If the continue button behaves like a link, it should be an action link

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24643]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

The continue button redirects the user to a clean link `https://staging.va.gov/health-care/health-questionnaires/questionnaires/answer-questions/veteran-information`. Because it behaves like a link, it should be styled and written as a link. In this case, [an action link may be the most semantic choice.](https://design.va.gov/components/action-links)

## Acceptance Criteria

- [ ] The continue button is styled and written as an action link

## Screenshots or Trace Logs

<img width="609" alt="Screen Shot 2021-05-12 at 6 45 42 PM" src="https://user-images.githubusercontent.com/14154792/118053628-e1af8a00-b352-11eb-8fe9-f538cb74f2e3.png">

**Comment by mdewey** [comment]
@joshkimux  This is a platform-level change. The button is in the code that is maintained by the platform team. The form code is located [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms/save-in-progress/FormStartControls.jsx#L96-L97). This would be a change to the `ProgressButton.` I am tagging @cvalarida for awareness and to get in his queue. This is a defect-3 so it doesn't need to be fixed now, but should be addressed.

**Comment by joshkimux** [comment]
Thanks @mdewey , appreciate that you've redirected this. For context @cvalarida , this isn't to suggest to change the `progressbutton` component itself, but that the "continue questions" pattern should use an `action link` component instead. In other words, I think there's a mismatch of components here since I believe "continue progress" in this instance is simply navigating the user into the form just like a "start xyz form" action link would (as opposed to saving a submission within a form when clicking the "continue" button there).
